pandas/io/excel/_odfreader.py

```diff
@@ -202,7 +202,8 @@ def _get_cell_value(self, cell, convert_float: bool) -> Scalar:
         elif cell_type == "time":
             result = pd.to_datetime(str(cell))
             result = cast(pd.Timestamp, result)
-            return result.time()
+            # error: Item "str" of "Union[float, str, NaTType]" has no attribute "time"
+            return result.time()  # type: ignore[union-attr]
```

Comment:
observation. `pd.Timestamp` resolves to `Any`, so cast above has no effect.

Reply:
yah, Timestamp is pretty much going to be left for last in this process.  Among other things, mypy complains about `__new__` sometimes returning NaT.